tests/common.py

```diff
@@ -338,7 +364,9 @@ def register_dummy_envs():
                                             gym.spaces.Box(
                                                 low=4, high=5, dtype=np.float32
                                             ),
-                                            gym.spaces.Text(1),
+                                            gym.spaces.Graph(
```

Comment:
Why the `Graph` space? Is Minari already compatible with this type of space?

Reply:
This list is used to test unsupported spaces; thus I removed `Text` and add `Graph`